src/lib/statistics/cardinality_estimator.hpp

```diff
@@ -31,23 +32,50 @@ class ValidateNode;
 class WindowNode;
 
 /**
- * Hyrise's default, statistics-based cardinality estimator.
+ * Hyrise's statistics-based cardinality estimator.
```

Comment:
Since you gathered a lot of knowledge, is there something more we would to say here as part of the overview?

Something brief about caching, diamonds, "instances" etc.?

Reply:
Added some comments here.